### PR TITLE
User list fixed to display all users

### PR DESCRIPTION
**Pull request by BobakOftadeh**
The user list now displays all users#5398

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/styles.scss [hunk 1]
```diff
@@ -40,3 +40,9 @@
   overflow: hidden;
   flex-shrink: 1;
 }
+
+
+.footer {
+  margin: 4rem
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/styles.scss]: Is missing a semicolon.

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/component.jsx [hunk 2]
```diff
@@ -257,6 +258,7 @@ class UserParticipants extends Component {
             <TransitionGroup ref={(ref) => { this.refScrollItems = ref; }}>
               { this.getUsers() }
             </TransitionGroup>
+            <div style={matchMedia(`only screen and (max-width: 640px)`).matches ? {margin: '4rem'}:{}}> </div>
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/component.jsx]: why don't use className? with className, you can use mq() and breakpoints.
E.G.: 
```
YOUR_CLASS_NAME {
	@include mq($small-only) {
		mergin: 4rem;
	}
}  
```

[Breakpoints](https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/stylesheets/variables/breakpoints.scss#L34)

**Comment by Tainan404** [comment]
Works in FF, but in Chrome it leaves an empty space.

empty-space

**Comment by BobakOftadeh** [comment]
Did you test it on a mobile device? The dev tools on chrome are not a correct representation as it does not have an address bar which would fill the whitespace.

**Pull request merged**